fairroot/base/sink/FairRootFileSink.cxx

```diff
@@ -68,6 +68,13 @@ FairRootFileSink::FairRootFileSink(const TString RootFileName, const char* Title
     LOG(debug) << "FairRootFileSink created------------";
 }
 
+FairRootFileSink::~FairRootFileSink()
+{
+    if (fRootFile) {
+        fRootFile->Close();
+    }
+}
+
```

Comment:
I think, this is not needed, as `fRootFile` is a `std::unique_ptr<TFile>` and so it is properly `delete`d (and the dtor is being called).

Reply:
But TFile needs both closed and deleted, right? I don't see anywhere saying file will be closed automatically when deleted.

Reply:
Ok, there is a Close method in the dtor of TFile.